src/fs-helpers/copyFile.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const fs = require('fs').promises;
```

Comment:
Current version of node.js docs recommend importing it like this:

```suggestion
const fs = require('node:fs/promises');
```

This has no real impact on app, but better to follow current [doc](https://nodejs.org/docs/latest-v18.x/api/fs.html).